### PR TITLE
docs, chore: add instruction for upgrade to `v1.0.1` and upgrade Sepolia

### PR DESCRIPTION
**Pull request by 0xHansLee**
No specific changes in flags, so it gives simple guide by changing image tag.
Also upgrade Sepolia client to `v1.0.1`.

### CODE REVIEW

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by ccrayz — COMMENTED** [review]
Shouldn't the docs for the v1.0.1 update state that it's for sepolia only?

**Comment by seolaoh** [comment]
> Shouldn't the docs for the v1.0.1 update state that it's for sepolia only?

We need to upgrade mainnet client to v1.0.1 also later, so I think it's ok to add it in advance now.

File: upgrades/v0.3.0.md [hunk 1]
```diff
@@ -9,10 +10,10 @@ This document provides instructions to upgrade from Kroma v0.2.2 to v0.3.0.
 For upgrade, stop your Kroma full node or validator.
 ```bash
 # for full node
-docker compose --profile fullnode down -v
+docker compose --profile fullnode down
```
Comment (wo-o) [inline upgrades/v0.3.0.md:13]: Can I know why the option `-v` removed?

**Comment by kimwoojin-kr** [comment]
> > Shouldn't the docs for the v1.0.1 update state that it's for sepolia only?
> 
> We need to upgrade mainnet client to v1.0.1 also later, so I think it's ok to add it in advance now.

I also think it should be announced after the mainnet updated to v1.0.1.

**Review by ccrayz — APPROVED** [review]

File: upgrades/v0.3.0.md (hunk 1 above)
Comment (seolaoh) [reply upgrades/v0.3.0.md:13]: As we discussed before, with `-v`, this command will remove the volume also. So we decided to remove the option as in `v1.0.0.md`, but it was not applied in `v0.3.0.md`, so I applied it.

**Review by wo-o — APPROVED** [review]

**Pull request merged**